example-ws/src/static/index.html

```diff
@@ -20,7 +20,9 @@
 
             const ws = new WebSocket("ws://127.0.0.1:8081");
             ws.onopen = (event) => {
-                rpc = new RpcBrowserWebSockets(ws);
+                // logger is optional third parameter, implementing interface IChildLogger:
+                // https://github.com/SAP/vscode-logging/blob/master/packages/types/api.d.ts#L17
+                rpc = new RpcBrowserWebSockets(ws /*, logger*/);
```

Comment:
```suggestion
                rpc = new RpcBrowserWebSockets(ws);
```

Reply:
why ? this is example, showing optional parameter (inline comment) and comment above explains the optional parameter type

Reply:
same as before

Reply:
done